engine/storage/datamotion/src/main/java/org/apache/cloudstack/storage/motion/StorageSystemDataMotionStrategy.java

```diff
@@ -920,6 +929,125 @@ private HostVO getHostOnWhichToExecuteMigrationCommand(VolumeInfo srcVolumeInfo,
         return hostVO;
     }
 
+    private VolumeInfo createTemporaryVolumeCopyOfSnapshotAdaptive(SnapshotInfo snapshotInfo) {
+        VolumeInfo tempVolumeInfo = null;
+        VolumeVO volumeVO = null;
```

Comment:
```suggestion
        VolumeVO tempVolumeVO = null;
```

Reply:
addressed in commit 42780e76163f72e7663b78945e74e62325fb4158 and 018e55678d494bdb53f1385fba57bd84031ae4a4